packages/starlight/components/CallToAction.astro

```diff
@@ -1,17 +1,19 @@
 ---
+import type { HTMLAttributes } from 'astro/types';
 import Icon from '../user-components/Icon.astro';
 import type { Icons } from './Icons';
 
 interface Props {
 	variant: 'primary' | 'secondary' | 'minimal';
 	link: string;
 	icon?: undefined | { type: 'icon'; name: keyof typeof Icons } | { type: 'raw'; html: string };
+	attrs?: HTMLAttributes<"a">;
```

Comment:
This doesn’t really impact frontmatter validation, but to be a bit more accurate, we should omit `href` from the type here as it is already set elsewhere:

```suggestion
	attrs?: Omit<HTMLAttributes<'a'>, 'href'>;
```

(I was also unsure about `class` given we‘re using `class:list` on the `<a>` element, but just tested and Astro does merge `class` and `class:list`, so should be fine.)